src/input/hotkey.c

```diff
@@ -160,10 +166,58 @@ static void input_number(int number)
     }
 }
 
+static void new_map(void)
```

Comment:
To be honest I'm not sure if this Ctrl+N hotkey is desirable: it puts you back to the main menu without warning whatsoever when in the city.

Reply:
Exactly my feeling when i got to test the ctrl-o/n keys yesterday.
Btw selecting new game from top menu had same effect, no warning and "suddenly" (ehm) in the main menu, maybe that should be improved?

Not sure if even shortcut for this is really needed/usefull. I have doubts about if someone wants to do 'new game/map' that often for it to deserve shortcut?

Reply:
It doesn't :D

Reply:
New game from the menu is a more explicit action than pressing a random key combination on the keyboard by accident, so I have no problem with that remaining like it is. I do think we can remove the Ctrl+N shortcut, like you mentioned I don't think it's used very often.